src/lib/openapi/spec/feature-search-query-parameters.ts

```diff
@@ -34,6 +34,18 @@ export const featureSearchQueryParameters = [
             'The state of the feature active/stale. The state can be specified with an operator. The supported operators are IS, IS_NOT, IS_ANY_OF, IS_NONE_OF.',
         in: 'query',
     },
+    {
```

Comment:
in the UI we'll mostly gonna use IS or IS_ANY_OF but there's not reason for the API not to support all of those